frontend/app/dtos/validate-address-response-dto.server.ts

```diff
@@ -0,0 +1,29 @@
+export interface ValidateWSAddressResponseDTOProps {
+  responseType?: string;
+  statusCode?: string;
+  functionalMessages?: {action: string, message: string}[];
+  message?: string;
+  warnings?: string;
+}
+
+export class ValidateWSAddressResponseDTO implements ValidateWSAddressResponseDTOProps {
+
+  responseType?: string;
+  statusCode?: string;
+  functionalMessages?: {action: string, message: string}[];
+  message?: string;
+  warnings?: string;
+
+  constructor(responseType?: string,
+    statusCode?: string,
+    functionalMessages?: {action: string, message: string}[],
+    message?: string,
+    warnings?: string) {
+
+    this.responseType = responseType;
+    this.statusCode = statusCode;
+    this.functionalMessages = functionalMessages;
+    this.message = message;
+    this.warnings = warnings;
+  }
+}
```

Comment:
Like the other classes, this isn't really necessary in JS.

Reply:
Moved response DTO to wsaddress-service and modified the return